### DISCUSSION TITLE
Is gerbera right for me?

### DISCUSSION THREAD

**Discussion by raba34** [gerbera/gerbera#1666] [Q&A]
Hello,

first of all, I would like to apologize for my perhaps somewhat naive question, but I am just starting to look into media servers.

So now my question: Can I configure gerbera so that users from the Internet can also access my videos like Youtube for example? So with thumbnails and with still images when scrolling or with copy protection?

If that does not work: what software for Ubuntu is there with which I can achieve this?

Thanks
Ralph

**Comment by KarlStraussberger** [comment]
The main focus of gerbera is to provide media in private networks for devices using UPnP/DLNA.

There is a web frontend to browse your media and modify some properties. You may forward it to the open web with reverse proxies, but this web frontend is far from beeing designed for different user roles and security.

If you want to share your files you may well look into NextCloud which has a different focus: file hosting and **document management**.

**Reply by raba34** [reply]
OK, I understand.

Thanks
Ralph
